app/helpers/govuk_design_system_helper.rb

```diff
@@ -46,8 +46,35 @@ def govuk_summary_list_entry(key, value, tag_options_key = {}, tag_options_value
     end
   end
 
+  def govuk_notification_banner(text, key = 'Important', tag_options = {})
```

Comment:
Pattern: I would suggest setting `nil` as the default for the `text` arg

Reply:
I would consider allowing this method to accept a content block as we may wish to add more than just a paragraph.

See: https://design-system.service.gov.uk/components/notification-banner/success/index.html

Reply:
Done and done